migrations/201709211605_transaction.js

```diff
@@ -1,10 +1,16 @@
 const setup = knex =>
   knex.schema.createTable('transaction', (t) => {
-    t.increments().index();
-    t.integer('event_id').unsigned().notNullable();
-    t.foreign('event_id').references('event.id');
-    t.integer('total').notNullable();
-    t.string('charge_id').notNullable();
+    t.increments()
+      .index();
+    t.integer('event_id')
+      .unsigned()
+      .notNullable();
+    t.foreign('event_id')
+      .references('event.id');
+    t.integer('total')
+      .notNullable();
+    t.string('charge_id')
+      .notNullable();
```

Comment:
@GameKyuubi Transaction table seems to refer to userId as FK. Is it missing here?
![2017-09-25 19 48 04](https://user-images.githubusercontent.com/28984604/30804984-7da1f1de-a22a-11e7-9762-2ee74e0a9628.png)